pkg/spdk/replica.go

```diff
@@ -1043,6 +1043,110 @@ func (r *Replica) Get() (pReplica *spdkrpc.Replica) {
 	return ServiceReplicaToProtoReplica(r)
 }
 
+func (r *Replica) Expand(spdkClient *spdkclient.Client, size uint64) error {
+	r.Lock()
+	defer r.Unlock()
+
+	r.log.Info("Expanding replica")
+
+	clusterSize, err := r.fetchClusterSize(spdkClient)
+	if err != nil {
+		return err
+	}
+
+	roundedSize := util.RoundUp(size, clusterSize)
+	if roundedSize != size {
+		return fmt.Errorf("replica %s rounded up spec size from %v to %v since the spec size should be multiple of MiB", r.Name, size, roundedSize)
+	}
+
+	if r.SpecSize > roundedSize {
```

Comment:
nit: You may just use `size` directly, as `roundedSize` is just for checking if `size` is equal to `roundedSize`. Using `size` and `roundedSize` in code might cause some confusion.